src/main/java/com/neilbaner/duke/task/TaskList.java

```diff
@@ -0,0 +1,35 @@
+package com.neilbaner.duke.task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList() {
+        tasks = new ArrayList<Task>();
+    }
+
+    public ArrayList<Task> getAllTasks() {
```

Comment:
Consider renaming to getTaskArray or getTaskList, to make it clear that the return of type ArrayList

Reply:
Done. Thank you.